test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -412,7 +412,6 @@ json_spirit::mObject fillBCTest(json_spirit::mObject const& _input)
 		}
 
 		blArray.push_back(blObj);  //json data
-		this_thread::sleep_for(chrono::seconds(1));
```

Comment:
I wonder why this line was here in the first place. have you tried to fill tests after this update?